CMakeLists.txt

```diff
@@ -333,13 +342,13 @@ if(TOP_LEVEL_PROJECT)
         endif ()
     endif(UNIX)
 
-    if(MAKE_R_ONLY)
+    if(PLUGIN)
         if("${packageManagement}" STREQUAL "RPM")
             SET_DEPENDENCIES(CPACK_RPM_PACKAGE_REQUIRES "hpccsystems-platform = ${CPACK_RPM_PACKAGE_VERSION}")
         elseif("${packageManagement}" STREQUAL "DEB")
-            SET_DEPENDENCIES(CPACK_DEBIAN_PACKAGE_DEPENDS "hpccsystems-platform = ${CPACK_RPM_PACKAGE_VERSION}")
+            SET_DEPENDENCIES(CPACK_DEBIAN_PACKAGE_DEPENDS "hpccsystems-platform (= ${CPACK_PACKAGE_VERSION})")
```

Comment:
@xwang2713 This code here will add a dependency to the platform.  Right now the plugins package is set to respect only the version of the platform that the plugin was built against.